pkg/webhook/federatedresourcequota/validating.go

```diff
@@ -164,3 +173,13 @@ func validateResourceList(resourceList corev1.ResourceList, fld *field.Path) fie
 
 	return errs
 }
+
+func validateFederatedResourceQuotaName(name string, fld *field.Path) field.ErrorList {
+	errs := field.ErrorList{}
+
+	if len(name) > labelValueMaxLength {
```

Comment:
Do you want to use this const `DNS1123LabelMaxLength`?
https://github.com/kubernetes/kubernetes/blob/8e3adc4df64d5b382c8916610313ce25e0df8e28/staging/src/k8s.io/apimachinery/pkg/util/validation/validation.go#L183

Reply:
already update to use this parameter. the above pkg already have been imported.